x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/index.ts

```diff
@@ -104,6 +112,10 @@ interface BaseOperationDefinitionProps<C extends BaseIndexPatternColumn> {
    * Should be i18n-ified.
    */
   displayName: string;
+  /**
+   * The default label is assigned by the editor
+   */
+  getDefaultLabel: (column: C, indexPattern: IndexPattern) => string;
```

Comment:
This probably needs the other columns as well (e.g. for cumulative sum we need to look up the name of the wrapped column)